homeassistant/components/config/core.py

```diff
@@ -21,7 +21,12 @@ class CheckConfigView(HomeAssistantView):
     @asyncio.coroutine
     def post(self, request):
         """Validate configuration and return results."""
-        errors = yield from async_check_ha_config_file(request.app['hass'])
+        hass = request.app['hass']
+
+        if hass.components.hassio.is_hassio():
```

Comment:
Instead of this, let's just call the service.

Reply:
Will not work, is a webview